### ISSUE TITLE
Unable to change JSX pragma

### ISSUE THREAD

**Issue by balajisivanath** [christianalfoni/webpack-bin#216]
Changing jsx pragma throws following error
![image](https://cloud.githubusercontent.com/assets/2368442/23006517/6129e38c-f428-11e6-8653-48bf8aa834f9.png)

**Comment by Guria** [comment]
webpackbin 2  is coming soon:
https://github.com/cerebral/webpackbin/
https://webpackbin-test.firebaseapp.com/

**Comment by developit** [comment]
The issue is still present in webpackbin 2.

**Comment by christianalfoni** [comment]
@developit Have you checked webpackbin.com ? Can not reproduce there. Or maybe some combination of stuff happens here, so need more details :)

**Comment by developit** [comment]
Yup - that's where I was editing.  On [this bin](https://www.webpackbin.com/bins/-KflCmJ5bvKsRF8WDkzb), try removing the `/** @jsx h */` comments - you'll get an error saying "React is undefined" because it's trying to use `React.createElement`.

**Comment by christianalfoni** [comment]
Ah, I see! Noted, will look into it as soon as I can :)
